packages/office-ui-fabric-react/src/components/Callout/CalloutContent.base.tsx

```diff
@@ -253,13 +254,15 @@ export class CalloutContentBase extends BaseComponent<ICalloutProps, ICalloutSta
   protected _dismissOnLostFocus(ev: Event) {
     const target = ev.target as HTMLElement;
     const clickedOutsideCallout = this._hostElement.current && !elementContains(this._hostElement.current, target);
+    const { preventDismissOnLostFocus } = this.props;
 
     if (
-      (!this._target && clickedOutsideCallout) ||
+      !preventDismissOnLostFocus &&
```

Comment:
This definitely shouldn't be here. Instead don't add the focus listener.

Reply:
Sure, I can fix that. I was following the pattern of `preventDismissOnScroll`. Should I give it the same treatment?

Also this is a backfix from #5092 , I can make a follow-up PR on master to do the same thing.

Reply:
Actually, looking into how the listeners are registered, is it correct to not register them based on `preventDismissOnLostFocus`? If `_addListeners` has already been called, according to the semantics of `_hasListeners`, it will not be called again. We could enter a situation where you create the Callout with `preventDismissOnLostFocus={true}`, later update it to `false`, but the handlers will not be registered.

Reply:
hmm. Yeah that's true. For now you're probably right. In the long term it should be streamlined in some way, or perhaps each type of dismiss should have it's own handler. So there would be onFocusDismiss and onScrollDismiss so the user could choose to pass or not pass those dismiss events through. But that should probably be for 7.0

For now keep things the way they are.

Reply:
@joschect do you approve this PR then? or are there changes needed?

Reply:
OWA has actually upgraded to Fabric 6 in the meantime, so unless we feel this API would be useful for any other Fabric 5 users, I can abandon this PR.